GTG/plugins/unmaintained/hamster/hamster.py

```diff
@@ -140,7 +142,7 @@ def sendTask(self, task):
         fact = activity
         if category:
             fact += f"@{category}"
-        fact += f",{description}{tag_str}"
+        fact += f",, {description}{tag_str}"
```

Comment:
I assume this is something that changed in hamster's API?

Reply:
yes, the description is separated with double comma

> v3.0 requires a double comma ,, to start a description (see #482). https://github.com/projecthamster/hamster/wiki/Entering-activities